src/chunk_constraint.c

```diff
@@ -906,3 +955,43 @@ chunk_constraint_rename_hypertable_constraint(int32 chunk_id, const char *oldnam
 													  RowExclusiveLock,
 													  CurrentMemoryContext);
 }
+
+int
+chunk_constraint_alter_hypertable_constraint(Oid chunk_relid, const char *hypertable_constraint_name, AlterTableCmd *cmd)
+{
+	Chunk	   *chunk = chunk_get_by_relid(chunk_relid, 0, true);
+	AlterHypertableConstraintInfo info = {
+		.base = {
+			.hypertable_constraint_name = hypertable_constraint_name,
+		},
+		.cmd = cmd,
+		.chunk_relid = chunk_relid,
+	};
+
+	return chunk_constraint_scan_by_chunk_id_internal(chunk->fd.id,
+													  chunk_constraint_alter_hypertable_tuple,
+													  hypertable_constraint_tuple_filter,
+													  &info,
+													  RowExclusiveLock,
+													  CurrentMemoryContext);
+}
+
+int
+chunk_constraint_validate_hypertable_constraint(Oid chunk_relid, const char *hypertable_constraint_name, AlterTableCmd *cmd)
```

Comment:
Again, this function is identical to the above one save for the tuple-handling function passed to the scan. You can use one function and pass in the tuple-handling function as a parameter instead.